.azuredevops/pipelines/ReleaseBuild.yml

```diff
@@ -333,22 +333,25 @@ stages:
           script: >
             mkdir $(Build.StagingDirectory)/NupackOutput;
             $configFilePath = "$(Pipeline.Workspace)/Extras/edk2-BaseCryptoDriver.config.json";
+            $licenseFile = "$(Pipeline.Workspace)/Extras/License.txt";
 
             Write-Host nuget-publish
             --Operation Pack
             --OutputLog "$(Build.StagingDirectory)/NugetPackagingLog.txt"
             --ConfigFilePath "$configFilePath"
             --InputFolderPath "$(Pipeline.Workspace)/Package"
             --Version "$(publish_version)"
-            --OutputFolderPath "$(Build.StagingDirectory)/NupackOutput";
+            --OutputFolderPath "$(Build.StagingDirectory)/NupackOutput"
+            --CustomLicensePath "$licenseFile";
```

Comment:
is the ; necessary?

Reply:
Yes.  It's required to mark the ending of the write-host command.